document.c

```diff
@@ -3358,6 +3358,65 @@ parse_atxheader(struct lowdown_doc *doc, char *data, size_t size)
 	return skip;
 }
 
+/*
```

Comment:
The definition for `html_find_block` was moved up to allow visibility in `html_find_end`.